File: main/src/main/java/net/fwitz/math/main/binary/complex/analysis/gamma/LnGammaPlot.java
package net.fwitz.math.main.binary.complex.analysis.gamma;

import net.fwitz.math.binary.complex.analysis.Gamma;
import net.fwitz.math.plot.binary.complex.color.DomainColoringHanlu;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class LnGammaPlot {
    public static void main(String[] args) {
        new ComplexFunctionPlot("lnGamma(z)")
                .computeFn(Gamma::lnGamma)
                .colorFn(new DomainColoringHanlu())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/analysis/gamma/LnGammaPlot.kt
package net.fwitz.math.main.binary.complex.analysis.gamma

import net.fwitz.math.binary.complex.analysis.Gamma
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot
import net.fwitz.math.plot.binary.complex.color.DomainColoringHanlu

object LnGammaPlot {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("lnGamma(z)")
        .computeFn(Gamma::lnGamma)
        .colorFn(DomainColoringHanlu)
        .render()
}